src/ThreeEditor/components/Dialog/OpenFileDialog.tsx

```diff
@@ -47,6 +47,7 @@ export function OpenFileDialog(props: OpenFileProps) {
 
 	return (
 		<Dialog
+			aria-label={'openProjectDialog'}
```

Comment:
Maybe rename it to `Toggle drawer`?

Reply:
this label is for the "open project" widget/pop-up

Reply:
I renamed "drawer button" to "toggle drawer button" though